data/yam/autoyast/textmode_all_extensions_s390x.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's rename to `*_all_modules_and_extensions_*` in all the files, long but worthy due to the confusion.